pagerduty/event_orchestration_cache_variable_test.go

```diff
@@ -0,0 +1,711 @@
+			"cache_variable": {
+				"id": "43f061e0-e92b-49f0-91a0-be79404319fc",
+				"name": "create_example",
+				"configuration": {
+					"type": "recent_value",
+					"source": "event.custom_details.error",
+					"regex": "[0-9]+"
+				}
+			}
+		}`))
+	})
+
+	resp, _, err := client.EventOrchestrationCacheVariables.CreateContext(context.Background(), oType, oId, input)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	want := &EventOrchestrationCacheVariable{
+		ID:   "43f061e0-e92b-49f0-91a0-be79404319fc",
+		Name: "create_example",
+		Configuration: &EventOrchestrationCacheVariableConfiguration{
+			Type:   "recent_value",
+			Source: "event.custom_details.error",
+			Regex:  "[0-9]+",
+		},
+	}
+
+	if !reflect.DeepEqual(resp, want) {
+		t.Errorf("returned \n\n%#v want \n\n%#v", resp, want)
+	}
+}
+
+func TestGlobalOrchestrationCacheVariableGet(t *testing.T) {
+	setup()
+	defer teardown()
+
+	oId := "d054a57d-5933-44ce-8173-8be0bbacdfff"
+	oType := "global"
+	id := "9aa13ae3-81f3-4456-9abc-79233555fc3f"
+	url := fmt.Sprintf("%s/%s/cache_variables/%s", eventOrchestrationBaseUrl, oId, id)
+
+	mux.HandleFunc(url, func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		w.Write([]byte(`{
+			"cache_variable": {
+				"id": "9aa13ae3-81f3-4456-9abc-79233555fc3f",
+				"name": "get_example",
+				"conditions": [{
+					"expression": "event.source matches part 'test'"
+				}],
+				"configuration": {
+					"type": "trigger_event_count",
+					"ttl_seconds": 70
+				},
+				"created_at": "2024-02-12T14:44:58Z",
+				"created_by": {
+					"id": "P8B9WR7",
+					"self": "https://api.pagerduty.com/users/P8B9WR7",
+					"type": "user_reference"
+				},
+				"updated_at": "2024-02-13T12:41:43Z",
+				"updated_by": {
+					"id": "P8B9WR8",
+					"self": "https://api.pagerduty.com/users/P8B9WR8",
+					"type": "user_reference"
+				}
+			}
+		}`))
+	})
+
+	resp, _, err := client.EventOrchestrationCacheVariables.GetContext(context.Background(), oType, oId, id)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	want := &EventOrchestrationCacheVariable{
+		ID:    "9aa13ae3-81f3-4456-9abc-79233555fc3f",
+		Name:  "get_example",
+		Conditions: []*EventOrchestrationCacheVariableCondition{
+			{
+				Expression: "event.source matches part 'test'",
+			},
+		},
+		Configuration: &EventOrchestrationCacheVariableConfiguration{
+			Type:       "trigger_event_count",
+			TTLSeconds: 70,
+		},
+		CreatedAt: "2024-02-12T14:44:58Z",
+		CreatedBy: &EventOrchestrationCacheVariableReference{
+			ID:   "P8B9WR7",
+			Self: "https://api.pagerduty.com/users/P8B9WR7",
+			Type: "user_reference",
+		},
+		UpdatedAt: "2024-02-13T12:41:43Z",
+		UpdatedBy: &EventOrchestrationCacheVariableReference{
+			ID:   "P8B9WR8",
+			Self: "https://api.pagerduty.com/users/P8B9WR8",
+			Type: "user_reference",
+		},
+	}
+
+	if !reflect.DeepEqual(resp, want) {
+		t.Errorf("returned \n\n%#v want \n\n%#v", resp, want)
+	}
+}
+
+func TestGlobalOrchestrationICacheVariableUpdate(t *testing.T) {
```

Comment:
Typo in `TestGlobalOrchestrationICacheVariableUpdate`
```suggestion
func TestGlobalOrchestrationCacheVariableUpdate(t *testing.T) {
```